test/version_sorter_test.rb

```diff
@@ -21,6 +21,10 @@ def test_sorts_versions_correctly
 
   def test_sorts_versions_like_rubygems
     versions = %w(1.0.9.b 1.0.9 1.0.10 2.0 3.1.4.2 1.0.9a 2.0rc2 2.0-rc1)
+    if (Gem.rubygems_version < Gem::Version.new('2.1.0')) 
+      # Old versions of RubyGems cannot parse semver versions like `2.0-rc1`
+      versions.pop()
```

Comment:
So rather than skip the test entirely, I just removed the offending version that cannot be parsed with older rubygems versions. So the test still runs and works - it just isn't as comprehensive for old versions.